candi/bashible/common-steps/node-group/031_install_containerd.sh.tpl

```diff
@@ -39,5 +39,12 @@ post-install() {
   fi
 }
 
+if [[ "$FIRST_BASHIBLE_RUN" == "yes" ]]; then
+  if command -v containerd >/dev/null 2>&1; then
+    bb-log-error "containerd exist on servers $HOSTNAME. Deckhouse not support working with embedded containerd. Please uninstall containerd and try again"
```

Comment:
"containerd is detected on $HOSTNAME. Deckhouse does not support pre-provisioned containerd installations. Please uninstall containerd and try again."